scripts/init.sql

```diff
@@ -8,14 +14,14 @@ CREATE TABLE "agencies" (
     "address_line_2" varchar(255),
     "city" varchar(100) NOT NULL,
     "state" varchar(100) NOT NULL,
-    "country" varchar(100) NOT NULL,
+    "country_code" varchar(100) NOT NULL,
     "zip_code" varchar(50) NOT NULL,
-    "verified" boolean NOT NULL DEFAULT false,
+    "is_verified" boolean NOT NULL DEFAULT false,
     "employer_identification_number" varchar(100) UNIQUE,
     "account_manager_id" uuid NOT NULL,
-    "image_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
-    "updated_at" timestamp
+    "image_id" uuid,
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP),
```

Comment:
changed all timestamps to `timestamptz` as I found great difficulty in getting Postgres to use the same timezone for everything regardless of system time, so it would be wise to store the timezone along with the timestamp for easier conversions in-code.